README.rst

```diff
@@ -14,17 +18,33 @@ Use pipenv to install and manage dependencies::
 
 Connecting to the data warehouse will require installing the ``cx_Oracle`` python package. The good news is that this is now being packaged as a wheel for most architectures, so no extra work is required to install it. If you don't need to actually connect to the data warehouse, you are done. Note that the test suite uses SQLite, so you can develop and test without connecting to the data warehouse.
 
-If you do need to connect to the data warehouse, you will also need to install the Oracle client library. It seems that now just installing the basic light package should be fine. In general, all you should need to do is extract the package and add the extracted directory to your ``LD_LIBRARY_PATH`` environment variable. If there is no ``lbclntsh.so`` (``libclntsh.dylib`` for Mac) symlink in the extracted directory, you will need to create one. The process will look something like this::
+If you do need to connect to the data warehouse, you have two options, one using Docker and one without.
+
+Without Docker
+^^^^^^^^^^^^^^
+
+To connect without Docker you will need to install the `Oracle client library <https://www.oracle.com/technetwork/database/database-technologies/instant-client/overview/index.html>`_. It seems that now just installing the basic light package should be fine. In general, all you should need to do is extract the package and add the extracted directory to your ``LD_LIBRARY_PATH`` environment variable. If there is no ``lbclntsh.so`` (``libclntsh.dylib`` for Mac) symlink in the extracted directory, you will need to create one. The process will look something like this (changing for paths/filenames as necessary)::
 
     $ unzip instantclient-basiclite-linux.x64-18.3.0.0.0dbru.zip -d /usr/local/opt
+
     # Add the following line to your .bash_profile or whatever to make it permanent
     $ export LD_LIBRARY_PATH=/usr/local/opt/instantclient_18_3:$LD_LIBRARY_PATH
+
     # If the symlink doesn't already exist:
     $ ln -rs /usr/local/opt/instantclient_18_3/libclntsh.so.18.1 \
         /usr/local/opt/instantclient_18_3/libclntsh.so
 
 On Linux, you will also need to make sure you have libaio installed. You can probably just use your system's package manager to install this easily. The package may be called ``libaio1``.
 
+With Docker
+^^^^^^^^^^^
+
+Connecting with Docker, in theory, should be more straightforward. The idea would be to test your changes in a container. As long as you aren't modifying the project dependencies, building the container should be quick, so iterating shouldn't be terrible. You will of course need a working Docker installation, and you will also need to have the AWS CLI installed and configured. Your development process using this method would look like:
+
+1. Make your changes.
+2. Run ``make dist`` from project root.
+3. Test your changes by running ``docker run --rm carbon <carbon args>``, with ``<carbon args>`` being whatever arguments you would normally use to run carbon.
+
```

Comment:
I can't comment on the bit I want to because it's not in this changes... under Building it mentions: 
```Running ``make dist`` creates a new container that is tagged as ``carbon:latest``.``` is that still accurate?

Reply:
Yeah